### ISSUE TITLE
Can't reproduce the performance of warmup(60k)

### ISSUE THREAD

**Issue by xsc1234** [microsoft/ANCE#22]
Hello, I used run_train_warmup.sh to train the warmup model and found that the performance of my model can not achieve the effect of your released checkpoint (pretrained BM25 warmup checkpoint MRR@10 is 0.311), even if I train it to 300k steps (MRR@10 is 0.2979).  All training hyperparameters are as described in run_train_warmup.sh, how can I handle this?
